.github/workflows/ci.yml

```diff
@@ -67,6 +67,27 @@ jobs:
           filter_mode: nofilter
           fail_on_error: true
 
+  code-style:
+    name: "Running code style checks"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Run PyAnsys code style checks"
+        uses: ansys/actions/code-style@v5.0
```

Comment:
We are already running `pre-commit` as you can see in the current PR for example: https://results.pre-commit.ci/run/github/434701261/1703862521.kJCxvLbnR6KdGiv51X3tHA

These "code-style" checks are just running `pre-commit`, as you can see in https://github.com/ansys/pyfluent/actions/runs/7357725428/job/20029817524?pr=2328

Reply:
I have removed the "code-style" check. Is there a benefit to using the `pre-commit-config.yaml` instead of "code-style"?

Reply:
As you can find in the `code-style` action documentation you linked above: https://actions.docs.ansys.com/version/stable/style-actions/index.html, this action also uses `.pre-commit-config.yaml`, so both of these options depend on this file